instrumentation/async-http-client/async-http-client-1.9/javaagent/src/test/java/io/opentelemetry/javaagent/instrumentation/asynchttpclient/v1_9/AsyncHttpClientTest.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.asynchttpclient.v1_9;
+
+import static io.opentelemetry.api.common.AttributeKey.stringKey;
+
+import com.ning.http.client.AsyncCompletionHandler;
+import com.ning.http.client.AsyncHttpClient;
+import com.ning.http.client.AsyncHttpClientConfig;
+import com.ning.http.client.Request;
+import com.ning.http.client.RequestBuilder;
+import com.ning.http.client.Response;
+import com.ning.http.client.uri.Uri;
+import io.opentelemetry.api.common.AttributeKey;
+import io.opentelemetry.instrumentation.testing.junit.InstrumentationExtension;
+import io.opentelemetry.instrumentation.testing.junit.http.AbstractHttpClientTest;
+import io.opentelemetry.instrumentation.testing.junit.http.HttpClientInstrumentationExtension;
+import io.opentelemetry.instrumentation.testing.junit.http.HttpClientResult;
+import io.opentelemetry.instrumentation.testing.junit.http.HttpClientTestOptions;
+import java.net.URI;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+import java.util.concurrent.ExecutionException;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+class AsyncHttpClientTest extends AbstractHttpClientTest<Request> {
+
+  @RegisterExtension
+  public static final InstrumentationExtension testing =
+      HttpClientInstrumentationExtension.forAgent();
+
+  private static final Long CONNECTION_TIMEOUT_MS = CONNECTION_TIMEOUT.toMillis();
+  private static final Long READ_TIMEOUT_MS = READ_TIMEOUT.toMillis();
```

Comment:
Nit: could be primitive values
```suggestion
  private static final long CONNECTION_TIMEOUT_MS = CONNECTION_TIMEOUT.toMillis();
  private static final long READ_TIMEOUT_MS = READ_TIMEOUT.toMillis();
```

Reply:
updated, thanks!